ex2_hal/sband/equipment_handler/include/sTransmitter.h

```diff
@@ -108,12 +108,12 @@
 // Conversion factors
 #define S_FREQ_OFFSET_SCALING 2
 #define S_FWVER_MAJORNUM_SCALING 100
-#define S_OUTPWR_SCALING 0.00114f
-#define S_PATEMP_SCALING 0.07324f
-#define S_PATEMP_OFFSET -50.0f
-#define S_TEMP_SCALING 0.0625f
-#define S_CURRENT_SCALING 0.00004f
-#define S_VOLTAGE_SCALING 0.004F
+#define S_OUTPWR_SCALING 114 / 100000
```

Comment:
Pretty sure these will all be 0. For example 4/5=0 in integer math

Reply:
These are used in very specific places and I made of the order of operations. Often a multiplication will happen on the first number, then the division

Reply:
This isn't really the standard way of doing it. It should be like this
```suggestion
#define S_OUTPWR_SCALING 114 / 100000
```#define S_OUTPWR_SCALING(n) ((n * 114) / 100000)

Reply:
Ah okay I'll fix that